longevity_test.py

```diff
@@ -78,6 +96,23 @@ def run_pre_create_keyspace(self):
         if self.params.get('pre_create_keyspace'):
             self._pre_create_keyspace()
 
+    def _run_validate_large_collections_in_system(self, node):
+        self.log.info("Verifying large collections in system tables on node: {}".format(node))
+        with self.db_cluster.cql_connection_exclusive(node=node) as session:
+            query = "SELECT * from system.large_cells WHERE keyspace_name='large_collection_test'" \
```

Comment:
nitpick, I would have not filter for a specific table, so this validation can be use on other test case if needed
also we don't expect other tables to have large cells, do we ?